snowplow/iglu-client-embedded/schemas/com.metabase/instance_stats/jsonschema/1-0-1

```diff
@@ -0,0 +1,212 @@
+{
+  "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "description": "Schema for daily stats ping, tracking instance metrics and settings",
+  "self": {
+    "vendor": "com.metabase",
+    "name": "instance_stats",
+    "format": "jsonschema",
+    "version": "1-0-1"
```

Comment:
After [discussion](https://metaboat.slack.com/archives/C04FZTRF457/p1728889142471029) with Joao, it seems a better and safer approach to go with `2-0-0`. Let's change this here and also rename the file.

Apart from that the file looks good to me. I also asked Joao to double-check whether it is valid (in the linked thread above)

Reply:
sounds good, done.